examples/list_disk_snapshots.py

```diff
@@ -64,8 +64,12 @@ def parse_args():
     # Create mapping from snapshot parent to snapshot, used to reconstruct the
     # snapshot chain.
     snapshots = {}
-    for s in disk_snapshots_service.list(include_active=True):
+
+    # Snapshots belonging to other disks will be added here
+    other_snapshots = []
+    for s in disk_snapshots_service.list(include_active=True, include_template=True):
```

Comment:
Is my understanding correct that the 1st loop is on items that match:
include_active=True
include_template=False
And the 2nd loop on items that match:
include_active=False
include_template=True

Reply:
It would work, but I preferred not to issue 2 requests and just looping over the snapshots that I didn't find. But actually Nir's solution is much better.